daml-lf/spec/transaction.rst

```diff
@@ -46,14 +49,30 @@ of this document, is not a valid DAML-LF transaction.
 Backward compatibility
 ^^^^^^^^^^^^^^^^^^^^^^
 
-DAML-LF transactions are accompanied by a version identifier; every
-change to ``transaction.proto`` entails a change to this specification,
-and every change to this specification introduces a unique new version.
-`Version history`_ defines a total ordering of all past versions; any
-version *y* unlisted there should be considered *y>k* for any known
-version *k*.
+DAML-LF transaction nodes, and transactions are encoded according a common versioning scheme, called the *transaction version scheme*.
+Each version of this scheme, called a transaction version, is associated to a unique language version.
+
+In the following *dev* transaction version will refer to a transaction version associated to a *dev* language version and
+*stable* transaction version will refer to a transaction version associated to a non-*dev* language version.
+
+Unlike the serialization format of a *dev* transaction version which can be changed freely without
+compatibility considerations, the serialization format of a *stable* transaction version cannot be changed
+or associated to a different language version once introduced.
+
+An consumer compliant with this specification does not have
+to provide any support for *dev* transaction version. It must however accept
+all stable versions.
+
+Every change to ``transaction.proto`` entails a change to this specification.
+`Version history`_ defines a total ordering of all past stable versions; any
+stable version *y* unlisted there should be considered *y>k* for any known
+version *k*.  At the top of the specification file, just under the title
```

Comment:
> [...]; any
stable version *y* unlisted there should be considered *y>k* for any known
version *k*.

I'm afraid I don't understand what you want to say here. Would you mind paraphrasing in a reply to this comment and I'll help to find a wording that we both understand then?

Reply:
This sentence is from the previous version originally written by @S11001001. (I just added "stable").
I understand that version not listed in the `Version history` should be considered newer than the one inside the `Version history` , in other words when decoding a  transaction, if you encounter a version you do not know anything about you should consider as newer than any one you know about.

Reply:
I see. That makes sense. Although I have a hard time reading it from there. Anyway, let's leave it as is for now.